src/platforms/apple/common/enriching-events/user-feedback.mdx

```diff
@@ -31,3 +31,27 @@ userFeedback.email = @"john.doe@example.com";
 userFeedback.name = @"John Doe";
 [SentrySDK captureUserFeedback:userFeedback];
 ```
+
+```csharp {tabTitle:C#}
+using Sentry;
+
+var eventId = SentrySdk.CaptureMessage("My message.");
+
+var userFeedback = new UserFeedback(
+  eventId,
+  "john.doe@example.com",
+  "It broke.",
+  "John Doe"
+);
+
+SentrySdk.CaptureUserFeedback(userFeedback);
```

Comment:
There's an extension method that takes the individual arguments, did I get it right?

Does it compile?

```suggestion
SentrySdk.CaptureUserFeedback(eventId,
  "john.doe@example.com",
  "It broke.",
  "John Doe");
```